jjb/edgex-go/edgex-go-snap.yaml

```diff
@@ -18,6 +18,8 @@
     jobs:
      - '{project-name}-{stream}-stage-snap-arm':
          build-node: ubuntu18.04-docker-arm64-4c-2g
+     - '{project-name}-release-snap':
```

Comment:
Given that there is no stream listed here, I assume this is meant to only run on master?

Reply:
Well technically the job doesn't run on any branch, it's meant more for _releasing_ an already built snap artifact. The snap artifacts are uploaded daily to the `edge`, `delhi/edge`, etc. channels and we manually promote individual artifacts from `edge` to `stable` or `delhi/stable`, etc. Previously this step was manual and required someone from LF to SSH into the machine and run a command using the credentials file discussed earlier.